src/main/java/dk/cngroup/lentils/service/ScoreService.java

```diff
@@ -51,4 +54,17 @@ public int getScoreByTeam(final Team team) {
                 .collect(Collectors.toList());
         return teamScores;
     }
+
+    public List<TeamScoreDetail> getTeamWithDetailScores(final Team team) {
+        List<TeamScoreDetail> teamScoreDetails = new ArrayList<>();
```

Comment:
Tohle by mozna slo kratseji prepsat pomoci streamu, ale to jen kdyby nahodou si vedela jak. Jinak supr.

Reply:
Já  jsem to zkoušela, ale pak jsem se do toho nějak zamotala a nechtěla jsem se zase moc zdržovat.. ale zkusím ještě jednou

Reply:
Jasne, tak to klidne nech, neva.